exemplar/src/main/kotlin/com/squareup/exemplar/ExemplarModule.kt

```diff
@@ -2,14 +2,18 @@ package com.squareup.exemplar
 
 import com.google.inject.AbstractModule
 import misk.web.WebActionModule
+import misk.web.actions.ReadinessCheckAction
 import misk.web.actions.InternalErrorAction
+import misk.web.actions.LivenessCheckAction
 import misk.web.actions.NotFoundAction
 
 class ExemplarModule : AbstractModule() {
     override fun configure() {
         install(WebActionModule.create<HelloWebAction>())
         install(WebActionModule.create<HelloWebPostAction>())
         install(WebActionModule.create<InternalErrorAction>())
+        install(WebActionModule.create<ReadinessCheckAction>())
```

Comment:
Do we want these added by default (I think so)

Reply:
Maybe, I don't know if we have too much in the way of defaults so far. Even `InternalErrorAction` isn't a default. I think in a followup, we can define a `DefaultWebActionsModule` which installs some sane defaults for users.

Reply:
yeah!